CHANGELOG.md

```diff
@@ -12,7 +12,33 @@ accidentally triggering the load of a previous DB version.**
 * #4397 Support intervals with month component when constifying now()
 
 **Thanks**
-@nikugogoi for reporting a bug with CTEs and upserts on distributed hypertables
+* @nikugogoi for reporting a bug with CTEs and upserts on distributed hypertables
+
+
+## 2.7.1 (2022-07-01)
+
+This release is patch release. We recommend that you upgrade at the
+next available opportunity.
+
+**Bugfixes**
+* #4244 Function telemetry
+* #4287 Add internal api for foreign table chunk
```

Comment:
Not a bugfix

Reply:
Yes, but it is an internal interface so it is safe to add into a patch release and we have requests for it internally.

Reply:
Moved it to the "Features" section instead.